examples/docker-pause.js

```diff
@@ -0,0 +1,6 @@
+import containers from 'k6/x/docker/containers';
+
+export default function () {
+  const containerId = 'YOUR_CONTAINER_ID';
+  containers.pauseContainer(containerId, {attach: true});
```

Comment:
```suggestion
  containers.pauseContainer(containerId);
```

implementation does not take any args

Reply:
✅